documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,255 @@ You may obtain a copy of the License at
+    read in the same order even if the Reader performs backs up and re-reads.
 
-3.  If an event has been acked to its writer or has been read by a reader it is guaranteed that it will continue to exist in the same place for all subsequent reads until it is deleted.
+- If an Event has been acknowledged to its Writer or has been read by a Reader, it is guaranteed that  it will continue to exist in the same location or position for all subsequent reads until it is deleted.
 
-4.  If there are multiple Readers reading a Stream and they all back up to any given point, they will never see any reordering with respect to that point. (It will never be the case that an event that they read before the chosen point now comes after or vice versa.)
+- If multiple Readers are reading a Stream and backs up for a while and then again when it performs re-reads, it is assured that no reordering would have happened.
 
-## ReaderGroup Checkpoints
+## Reader Group Checkpoints
 
-Pravega provides the ability for an application to initiate a Checkpoint on a
-ReaderGroup.  The idea with a Checkpoint is to create a consistent "point in
-time" persistence of the state of each Reader in the ReaderGroup, by using a
-specialized Event (a Checkpoint Event) to signal each Reader to preserve its
-state.  Once a Checkpoint has been completed, the application can use the
-Checkpoint to reset all the Readers in the ReaderGroup to the known consistent
+Pravega provides the ability for an application to initiate a **Checkpoint** on a
+Reader Group.  The idea with a Checkpoint is to create a consistent "point in
+time" persistence of the state of each Reader in the Reader Group, by using a
+specialized Event (_Checkpoint Event_) to signal each Reader to preserve its
+state. Once a Checkpoint has been completed, the application can use the
+Checkpoint to reset all the Readers in the Reader Group to the known consistent
 state represented by the Checkpoint.
 
-For more details on working with ReaderGroups, 
-see [ReaderGroup Basics](basic-reader-and-writer.md#readergroup-basics).
+For more details on working with Reader Groups, Please see [Reader Group Basics](basic-reader-and-writer.md#readergroup-basics).
 
 
 ## Transactions
 
-Pravega supports Transactions.  The idea of a Transaction is that a Writer can
-"batch" up a bunch of Events and commit them as a unit into a Stream.  This is
-useful, for example, with Flink jobs, using Pravega as a sink.  The Flink job
-can continuously produce results of some data processing and use the Transaction
-to durably accumulate the results of the processing.  At the end of some sort of
-time window (for example) the Flink job can commit the Transaction and therefore
+Pravega supports Transactions. The idea of a transaction is that a Writer can
+"batch" up a bunch of Events and commit them as a unit into a Stream. This is
+useful, for example, in Flink jobs, Pravega is used as a sink.  The Flink job
```

Comment:
Please, change: `in Flink jobs, Pravega is used as a sink` -> `in Flink jobs using Pravega as a sink`